CONTRIBUTING.md

```diff
@@ -14,13 +14,23 @@ Contributions are more than welcome. If you want to contribute, please make sure
 Before creating a Pull Request you should validate your code with ESLint.
 
 ```
-npm run eslint
+npm run lint
 ```
 
 ## Developing with Docker
 
 In case you want to quickly build a docker image locally to check if it works then run `npm run docker-build`
 
+In case you don't have a local node environment you can run 
+
+```
+docker build -t asyncapi/generator:latest .
+docker run -v `pwd`:`pwd` -w `pwd` -it --rm --entrypoint /bin/sh  asyncapi/generator:latest
+npm install
+npm run lint
```

Comment:
I understand why you put `npm run lint` command, but maybe there we shouldn't write about it? `Developing with Docker` section should be point only for developing, not to action before creating PR. If you have a different opinion on this, feel free to discuss.

Reply:
I have the feeling, that a simple quick start for contribution is helpful. Not all contributers are node developers. Where would be a better location in this project?